apps/docs/src/app/platform/component-docs/platform-menu-button/platform-menu-button-header/platform-menu-button-header.component.html

```diff
@@ -1,10 +1,21 @@
 <header>Menu Button</header>
 <description>
-    Menu Button works as Dropdown.
-    <br />
-    It selects from Menu whichs appears on Menu button click.
+    The menu button is used for triggering the display of a menu. <br />
+    In ngx-core, it is an extenstion of ngx-button directive, but in Platform this is a separate component.
+    <h3>Platform specific enhancements:</h3>
+    <ul>
+        <li>Menu Button wrapped up as a component.</li>
+        <li>
+            This component is used in conjunction with the
+            <a href="https://sap.github.io/fundamental-ngx/#/platform/menu">Platform Menu Trigger</a>.
+        </li>
+        <li>
+            Menu management is made easy by abstracting the menu definition and linking it with
+            <a href="https://sap.github.io/fundamental-ngx/#/platform/menu">Menu Trigger</a>.
+        </li>
+    </ul>
 </description>
-<import module="PlatformMenuButtonModule"></import>
+<pre style="background-color: #e9e7e7;"><import module="PlatformMenuButtonModule"></import></pre>
```

Comment:
Are all pages using the `<pre>` tag with a background color? We should be consistent in our use of markup in the documentation. Unless all pages are doing the same thing I would change this back.

Reply:
It will affect the platform look, Ideally, the look and feel of the platform should be the same as the core.

Reply:
removed pre tag. as other component does not have same convention. and it differs from core component pages.